Xabe.FFmpeg/FFprobeWrapper.cs

```diff
@@ -76,6 +81,18 @@ private int GetGcd(int width, int height)
             return width == 0 ? height : width;
         }
 
+        public async Task<T> Start<T>(string args)
+        {
+            args = "-print_format json " + args;
```

Comment:
What if in args will be another -print_format?
And what if user don't want this serialized

Reply:
If don't want deserialization should use raw result form of this function. Always is getting last "-print_format" parameter - if somone choose this option and pass another format deserialization will break. This function is only helper for typical usage.